Core/Services/StitchCoreRemoteMongoDBService/Sources/StitchCoreRemoteMongoDBService/Sync/BSONFixes.swift

```diff
@@ -0,0 +1,39 @@
+import MongoSwift
+
+func validateBSONTypes(_ lhs: BSONValue, _ rhs: BSONValue) {
```

Comment:
[nit] maybe add TODOs in this file referring to the `SWIFT-` tickets that they are waiting on